package.json

```diff
@@ -6,12 +6,8 @@
   },
   "name": "d2l-inputs",
   "scripts": {
-    "lint": "npm run lint:wc && npm run lint:js",
-    "lint:js": "eslint . --ext .js,.html test/**/*.html demo/**/*.html",
-    "lint:wc": "polymer lint",
```

Comment:
Is it still worth linting the remaining Polymer style files?

Reply:
Yeah, I was probably a bit hasty.  😏